registry/consumer/service_event_consumer.py

```diff
@@ -164,13 +164,13 @@ def _process_service_data(self, org_id, service_id, service_metadata, tags_data,
 
         if not existing_service:
             self._service_repository.add_service(recieved_service, BLOCKCHAIN_USER)
-        elif existing_service.service_state.transaction_hash == transaction_hash:
-            self.___curate_service_in_marketplace(service_id, org_id, curated=True)
         elif existing_service.service_state.transaction_hash is None:
             self._service_repository.save_service(BLOCKCHAIN_USER, existing_service, ServiceStatus.DRAFT.value)
         elif existing_service.service_state.transaction_hash != transaction_hash:
             # TODO:  Implement major & minor changes
             self._service_repository.save_service(BLOCKCHAIN_USER, existing_service, ServiceStatus.DRAFT.value)
+        elif existing_service.service_state.transaction_hash == transaction_hash:
```

Comment:
doest make sense from what I see

Reply:
@rajeevbbqq has taken care of the order after discussing it with @anandrgit